test/test_events.py

```diff
@@ -314,3 +319,18 @@ def test_events_filter_itee(uqcsbot: MockUQCSBot):
     assert len(messages) == 2
     expected = "_Events in the next *2 weeks*:_"
     assert messages[1].get('text') == expected
+
+@patch("uqcsbot.utils.itee_seminar_utils.get_seminar_summary_page",
+       new=mocked_html_summary_get_no_results)
+@patch("uqcsbot.scripts.events.get_calendar_file", new=mocked_events_ics)
+@patch("uqcsbot.scripts.events.get_current_time", new=mocked_get_august_2018_time)
+def test_events_recurring(uqcsbot: MockUQCSBot):
+    """
+    This test simulates the user invoking '!events', for both UQCS and Seminar calendars
```

Comment:
May also be worth clarifying that the point here is solely to test recurring events.

Reply:
Whoops i forgot to change the docstring from the function above